src/api/services/timecardService.spec.js

```diff
@@ -17,6 +17,17 @@ const timecard = [
   },
 ];
 
+const TIME_PERIOD_TYPES = {
+  meta: {
+    next: null,
+  },
+  items: [
+    {
+      name: 'Scheduled rest day',
+    },
+  ],
+};
+
```

Comment:
Could you import `timeCardPeriodTypes` from the mock data file and test that instead of creating a new variable here?

Reply:
Good point. Incorporated